### ISSUE TITLE
Open Visualization Session Proposal: deck.gl, GeoArrow and Lonboard

### ISSUE THREAD

**Issue by kylebarron** [openjs-foundation/summit#410]
### Proposal

This is just a stub, but I propose to give a talk on the architecture of [Lonboard](https://developmentseed.org/lonboard/latest/), how it uses deck.gl, and how we can better integrate [GeoArrow](https://geoarrow.org/) into deck.gl in the future.

There's some initial Arrow-deck.gl bindings in https://github.com/geoarrow/deck.gl-layers / https://github.com/visgl/deck.gl-community/tree/master/modules/arrow-layers

**Topic of the session**

Introducing GeoArrow, how GeoArrow can improve performance in deck.gl, how Lonboard makes use of GeoArrow and deck.gl to provide a fast visualization library for Python users.

**Type of the session**

- [ ] Collaborate
- [ ] Workshop
- [x] Talk

**Estimated duration of the session**

30 min?

**Date and Time of the session**

**Level**

- [ ] Beginner
- [x] Intermediate
- [x] Advanced

**Pre-requisite knowledge**

**Describe the session**

**Session facilitator(s), Github handle(s) and timezone(s)**

**Meeting notes and Virtual Meeting Link**

**Follow-up / Set-up sessions (if any)**

**Additional context (optional)**

----

**Comment by chrisgervang** [comment]
Hey Kyle! Thank you for your submission. If you have a chance, could you help us flush this out some more with how much time you like, and an abstract? We're hoping to have all submissions by July 10th so we can put together an agenda. Thanks!

Also, just confirming you're expecting to attend in person sine we aren't able to support remote sessions this year. Looking forward to seeing you there!

**Comment by kylebarron** [comment]
Yes I'll be there in person. I figure I can fit into whatever the normal session time is, maybe 30 minutes?

**Comment by chrisgervang** [comment]
Hey Kyle, we've scheduled your talk on Day 1 at 11:30 for 30 minutes, and you can view it in our [draft agenda](https://docs.google.com/document/d/18PnITkoJAaxbLuyGIBwdS7wwfrJnWoJUaJvvarF_6Xg/edit). 

We'd like to feature your picture and title on our event website, similar to what we did last year ([see last year's event page](https://deck.gl/events/new-york-summit-2023/) for reference). If you’re interested, please send us a recent photo and your current title.

**Comment by kylebarron** [comment]
![_R3_2305 (1)](https://github.com/user-attachments/assets/48ef9304-a8a1-4848-bcfb-ed2791cec65d)

Here's a headshot. And current title is Cloud Engineer at Development Seed

**Comment by Pessimistress** [comment]
The latest event agenda is posted [here](https://deck.gl/events/london-summit-2024/index.html). Please let me know if you need to update your picture/title/affiliation.
I am looking forward to your talk!

**Comment by joyeecheung** [comment]
The summit has passed. closing.
